### DISCUSSION TITLE
DateTimeRange 国际化的问题

### DISCUSSION THREAD

**Discussion by ysjr-2002** [dotnetcore/BootstrapBlazor#3793] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

基于Bootstrapblazor6.5版本做国际化过程中遇到一个问题，
services.AddRequestLocalization<IOptions<BootstrapBlazorOptions>>((localizerOption, blazorOption) =>
{
      var cultures = new[] { "zh-CN" };
      localizerOption.AddSupportedCultures(cultures);
      localizerOption.AddSupportedUICultures(cultures);
});

页面使用DateTimeRange控件，<DateTimeRange DateFormat="yyyy-MM-dd" @bind-Value="@context.Range"></DateTimeRange> 当指定简体中文时工作正常。
但指定英文时
services.AddRequestLocalization<IOptions<BootstrapBlazorOptions>>((localizerOption, blazorOption) =>
{
      var cultures = new[] { "en-US" };
      localizerOption.AddSupportedCultures(cultures);
      localizerOption.AddSupportedUICultures(cultures);
});
出现错误以下错误
<img width="1066" alt="1720170428809" src="https://github.com/dotnetcore/BootstrapBlazor/assets/7730299/c110d862-d57c-4963-8c95-1e089bc57109">

请问是什么原因引起的错误？

### .NET Version

5.0

### Anything else?

BootstrapBlazor版本是6.5，操作系统Windows 11

**Comment by bb-auto** [comment]
@ysjr-2002 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@ysjr-2002 感谢提交 Issue 以我多年的经验我猜是时间格式化设置有问题

**Reply by ysjr-2002** [reply]
感谢你的回复。能协助解决吗？这是一个2年前开发的项目。现在客户希望能支持英文。测试后其它控件都运行正常，但DateTimeRange控件异常。

**Reply by ArgoZhang** [reply]
可加我微信协商
